package.json

```diff
@@ -32,14 +32,14 @@
   },
   "dependencies": {
     "debug": "^4.3.2",
-    "hof": "^20.4.0",
+    "hof": "~21.0.0",
     "hogan.js": "^3.0.2",
     "is-pdf": "^1.0.0",
     "jquery": "^3.6.0",
     "lodash": "^4.17.21",
     "mixwith": "^0.1.1",
     "moment": "^2.29.4",
-    "notifications-node-client": "^6.0.0",
+    "notifications-node-client": "^8.0.0",
```

Comment:
so this looks like it uses gov notify separately, should we create a ticket so that it uses it directly from hof?

Reply:
Thanks for your thoughts, I've now created a ticket.  I'll resolve this